Makefile

```diff
@@ -72,9 +72,9 @@ publish:
 
 bootstrap:
 	make clean-all
-	npm install
+	yarn
 	./node_modules/.bin/lerna bootstrap
 	make build
 	cd packages/babel-runtime; \
-	npm install; \
```

Comment:
Not sure what we should do here - actually wondering why bootstrap doesn't cover `npm install` in babel-runtime anyway..